src/buildtool.ts

```diff
@@ -1,12 +1,13 @@
 // Copyright 2022-2024 The MathWorks, Inc.
+import * as path from "path";
 
 export interface RunBuildOptions {
     Tasks?: string;
     BuildOptions?: string;
 }
 
 export function generateCommand(options: RunBuildOptions): string {
-    let command: string = "buildtool";
+    let command: string = "addpath('"+ path.join(__dirname, "plugins") +"');buildtool"
```

Comment:
Still need to escape single quotes.
```suggestion
    const pluginsPath = path.join(__dirname,"plugins").replace("'","''");
    let command: string = "addpath('"+pluginsPath+"');buildtool";
```

Reply:
done